group_vars/all

```diff
@@ -32,3 +32,13 @@ maas_postgres_primary_uri: "postgres://{{ maas_postgres_uri_login }}@{{ maas_pos
 maas_postgres_uri_snap: "maas-test-db:///"
 maas_alt_uri: "{{ maas_postgres_primary_uri if maas_postgres_installation_type|default(maas_installation_type)|lower == 'deb' else maas_postgres_uri_snap }}"
 maas_postgres_uri: "{{ maas_postgres_proxy_uri if maas_proxy_postgres_proxy_enabled else maas_alt_uri }}"
+
+maas_postgres_backup_dir: "/tmp/maas_backup/" 
+maas_postgres_backup_path: "{{ maas_postgres_backup_dir }}dump.sql"
```

Comment:
instead of dumping to a `.sql` file, did you consider `.sql.gz` or similar? as per https://docs.ansible.com/ansible/latest/collections/community/postgresql/postgresql_db_module.html#parameter-state

Reply:
we could, and I'd prefer to do so, I was just doing as our docs suggest. I'll update to `.sql.gz`